utils/email-service.js

```diff
@@ -13,8 +13,8 @@ function waitForEmail (recipientEmail) {
   })
 }
 
-function generateRandomEmailAddress () {
-  return client.servers.generateEmailAddress(serverID)
+function generateEmailAddress (subdomain = 'xyz') {
+  return `${subdomain}.${process.env.EMAIL_SERVICE_SERVER_ID}@mailosaur.io`
```

Comment:
What's the reason for this change?

Reply:
A few reasons:
1) It was named "generateRandom.." - it wasn't actually random
2) Because it used the same email address each time, when more than one test was sending emails, sometimes the wrong email was being retrieved.
3) By being able to change the email address per spec, we can ensure that we are only retrieving the correct emails.

You can see how we use this in the various specs, eg: `generateEmailAddress('save-form')`

Reply:
👍